src/types.ts

```diff
@@ -149,6 +152,13 @@ export enum TransferResult {
   FORCE_VALID,
 }
 
+export enum BallotStage {
+  Prep = 'PREP',
```

Comment:
Enum values should normally be in PascalCase (`'PREP'` -> `'Prep'`) unless they NEED to be in upper case (for example if they are returned that way from the smart contracts and you're using the enum directly to compare against them)